src/lib.rs

```diff
@@ -209,15 +219,14 @@ pub use rand_core::{RngCore, BlockRngCore, CryptoRng, SeedableRng};
 pub use rand_core::{ErrorKind, Error};
 
 // external rngs
-pub use jitter::JitterRng;
-#[cfg(feature="std")] pub use os::OsRng;
+#[doc(inline)] pub use jitter::JitterRng;
+#[cfg(feature="std")]
+#[doc(inline)] pub use os::OsRng;
```

Comment:
Keep the OsRng import on a single line please; this causes chaos with diff tools.